### DISCUSSION TITLE
This is the only solution that works

### DISCUSSION THREAD

**Discussion by shimdidly** [sshuttle/sshuttle#835] [General]
I tried OpenVPN, Wireguard, udp2raw, updtunnel, badvpn with ssh. Nothing worked, and the configuration was complicated.

When I was at my wits end, I found sshuttle. It was the simplest of all the solutions (a single command) and it not only works, it works really well. It's fast!!

Seriously, thank you so much.

**Comment by qadzek** [comment]
I would like to repeat that sentiment. I was in a situation where all VPN's were blocked, even the VPN server at my home didn't work. SSH wasn’t blocked though, so sshuttle saved the day.

Thanks a ton!
